prow/plugins/releasenote/releasenote.go

```diff
@@ -237,6 +251,16 @@ func handlePR(gc githubClient, log *logrus.Entry, pr *github.PullRequestEvent) e
 		}
 	}
 
+	if labelToAdd == releaseNoteNone && prLabels.Has(deprecationNote) {
+		labelToAdd = ReleaseNoteLabelNeeded
```

Comment:
This seems wrong.. why would we overwrite this, as opposed to adding some logic to `determineReleaseNoteLabel`?

Reply:
Thank you for the review. Fixed.